crates/ubrn_common/src/rust_crate.rs

```diff
@@ -88,6 +88,31 @@ impl CrateMetadata {
     }
```

Comment:
This refactor used to be relevant to this PR, but now we're using `ci.crate_name()` it is not. 

It is still a useful refactor, but we're in a high velocity phase of development, so would rather not spend time picking it out into a separate PR.

WDYT? should I leave it here, or break it out?

Reply:
My opinion, leave it in.  

I've generally found that trying to separate out changes after the fact leads to more problems than not, but ultimately your call.